Gopkg.toml

```diff
@@ -186,4 +186,4 @@ required = [
 
 [[prune.project]]
   name = "github.com/giantswarm/k8scloudconfig"
-  unused-packages = false
+  unused-packages = false
```

Comment:
Can you fix this with your editor to have a newline at the end of files.

Reply:
Done.